doc/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docs/base:latest
+MAINTAINER Sven Dowideit <SvenDowideit@docker.com> (@SvenDowideit)
+
+# to get the git info for this repo
+COPY . /src
+
+# Reset the /docs dir so we can replace the theme meta with the new repo's git info
+RUN git reset --hard
+
+RUN grep 'var Version' /src/version/version.go  | sed 's/.*"\(.*\)".*/\1/' > /docs/VERSION
```

Comment:
this is consistent with what the other projects are doing, is it good enough for a first pass (which you can change)